tests/fast/ShardedRocksNondeterministicTest.toml

```diff
@@ -0,0 +1,48 @@
+[configuration]
+storageEngineType = 5 # Always pick ShardedRocks
```

Comment:
Reason for having a separate sharded rocks non deterministic test:
1. We have exclude storage engine list parameter, but not include list parameter. Can be added but because of (2) I wanted a separate test anyway.
2. Having two separate tests hopefully increases the probability that we exercise metrics code (which is different for rocksdb and sharded rocks).